src/VisualStudio/Core/Def/EditorConfigSettings/SettingsEditorFactory.cs

```diff
@@ -175,5 +175,17 @@ public int MapLogicalView(ref Guid rguidLogicalView, out string? pbstrPhysicalVi
                 return VSConstants.E_NOTIMPL;   // you must return E_NOTIMPL for any unrecognized rguidLogicalView values
             }
         }
+
+        public object? GetDocumentData(uint grfCreate, string pszMkDocument, IVsHierarchy pHier, uint itemid)
+            => null;
```

Comment:
👍 Based on the editor's usage of this method, I can confirm returning `null` is the same as not implementing `IVsEditorFactory4`